client/post-editor/editor-ground-control/history-button.jsx

```diff
@@ -33,5 +50,6 @@ export default flow(
 	localize,
 	connect( null, {
 		openDialog: openPostRevisionsDialog,
+		recordTracksEvent,
```

Comment:
what if instead of these two props we passed in a new one called `selectRevision` (now I don't know if this is the right naming from the customer's perspective or from this component's perspective, but it looks like a history button might be used to select revisions…

```js
connect( null, {
  selectRevision: withAnalytics( recordTracksEvent( 'history_button_clik' ), openPostRevisionsDialog )
} )( HistoryButton )
```

then in our component instead of calling `this.onClick` we just call the action that makes sense from this component's perspective

```js
<button onClick={ this.props.selectRevision }>
```

Reply:
Thanks for the tips!